postgis/postgis.sql.in

```diff
@@ -2860,6 +2860,18 @@ CREATE OR REPLACE FUNCTION ST_Transform(geom geometry, from_proj text, to_srid i
 	LANGUAGE 'sql' IMMUTABLE STRICT PARALLEL SAFE
 	_COST_HIGH;
 
+CREATE OR REPLACE FUNCTION postgis_spatial_ref_sys(auth_name_in text, auth_srid_in text, OUT auth_name text, OUT auth_srid TEXT, OUT srtext TEXT, OUT proj4text TEXT)
+	RETURNS RECORD
```

Comment:
How about RETURNS spatial_ref_sys ?

Reply:
I'm trying to avoid deps on types in the database, deliberately, hence reading the TupleDesc off the function signature.

Reply:
Honestly, it feels overdesigned to me, very YAGNI.

Reply:
Oh and if we go with allowing people to specify the auth_name and auth_srid, then we would have one signature:

```
CREATE OR REPLACE FUNCTION postgis_spatial_ref_sys(auth_name_in text DEFAULT NULL, auth_srid_in text DEFAULT null)
	RETURNS TABLE (auth_name text, auth_srid TEXT, srtext TEXT, proj4text TEXT)
```
if no args then return everything and if args passed filter based on input args.